wi-fi/whd/whd_chip.c

```diff
@@ -612,11 +612,9 @@ whd_result_t whd_ioctl_print(whd_driver_t whd_driver)
 				data++;
 			}
 
-			if (strlen((char *)data) <= WHD_IOVAR_STRING_SIZE)
-				strncpy(iovar, (char *)data, strlen((char *)data));
+			snprintf(iovar, sizeof(iovar), "%s", data);
```

Comment:
strncpy?

Reply:
I've used snprintf to shut down compiler warning.